### ISSUE TITLE
RHC git-clone doesn't work on latest 2.9.0.windows.1 git version (Windows 10)

### ISSUE THREAD

**Issue by alisiikh** [openshift/rhc#758]
Hello, I'm using the latest version of Git for Windows 2.9.0.windows.1

The program is installed in directory: C:\Users\%USERNAME%\AppData\Local\Programs\Git

It seems that RHC is unable to find it, though Git's bin folder is added to %PATH% variable and can be accessed from any cmd client.

Message from RHC:
```
You do not have git installed. In order to fully interact with OpenShift you
will need to install and configure a git client. We recommend this free
application: Git for Windows - a basic git command line and GUI client
http://msysgit.github.io/.
```

It looks like the problem is in https://github.com/openshift/rhc/blob/master/lib/rhc/git_helpers.rb#L24-L33

**Comment by seshaljain** [comment]
@alisiikh, most probably you do not have Git on your your `PATH`. Check and do tell if you have it on your `PATH`.

**Comment by alisiikh** [comment]
Nope, basically the %PATH% was set up correctly, but for some reason I got a bug when not all env-vars got resolved by cmd client. So it was like 50/50 that git was found or not, for the first time PATH was fully resolved, and the other time it might be not resolved well, so that there are env-vars which are left unwrapped.

Looks like the problem is with my Windows system, so I just close this issue.
